samples/snippets/sample_templates.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Iterable
+
+from google.cloud import compute_v1
+
+
+def get_instance_template(project_id: str, template_name: str) -> compute_v1.InstanceTemplate:
+    """
+    Retrieve information about given instance template.
+
+    Args:
+        project_id: project ID or project number of the Cloud project you use.
+        template_name: name of the template you want to get.
+
+    Returns:
+        InstanceTemplate object containing information about given template.
+    """
+    template_client = compute_v1.InstanceTemplatesClient()
+    return template_client.get(project=project_id, instance_template=template_name)
+
+
+def list_instance_templates(project_id: str) -> Iterable[compute_v1.InstanceTemplate]:
+    """
+    Get a list of InstanceTemplate objects available in given project.
+
+    Args:
+        project_id: project ID or project number of the Cloud project you use.
+
+    Returns:
+        Iterable list of InstanceTemplate objects.
+    """
+    template_client = compute_v1.InstanceTemplatesClient()
+    return template_client.list(project=project_id)
+
+
+def create_template(project_id: str, template_name: str) -> compute_v1.InstanceTemplate:
+    """
+    Create a new template in given project with provided name.
+
+    Args:
+        project_id: project ID or project number of the Cloud project you use.
+        template_name: name of the new template that is created.
+
+    Returns:
+        InstanceTemplate object describing the new instance template.
```

Comment:
Should this say "...object that represents the..." instead? It's a little unclear why it would only describe the template rather than _be_ the template itself.

Reply:
You're right, represents is better.